ember-headless-form/src/components/-private/control/input.hbs

```diff
@@ -1,6 +1,6 @@
 <input
   type={{@type}}
-  value={{this.valueAsString}}
+  value={{@value}}
   id={{@fieldId}}
   ...attributes
```

Comment:
Question: what happens if someone adds their own `{{on 'input' this.handleInput}}` to this element? Or is this a private component that won't be used directly by the consumer?

Reply:
it would layer on top of this `{{on 'input'}}` -- so there would be two event listeners.

We want the element's API to be public, too

Reply:
> Or is this a private component that won't be used directly by the consumer?

We yield the component, so yes, a user could do `<field.input {{on "input" this.doSomething}}/>`. But as Preston said, you just end up with two separate event listeners. Which should be able to peacefully live together...

Unless one of those does something like `event.stopImmediatePropagation()`, but then you are basically begging for problems! 😅